src/Console/ConsoleOptionParser.php

```diff
@@ -774,7 +774,7 @@ protected function _optionExists(string $name): bool
         if (str_starts_with($name, '--')) {
             return isset($this->_options[substr($name, 2)]);
         }
-        if ($name[0] === '-' && $name[1] !== '-') {
+        if (str_starts_with($name, '-')) {
```

Comment:
Isn't this a bit different now? Previously the code was testing for one, but not two `-` as that indicates a short option. The new logic only checks for a single `-`.

Reply:
The case for two `-` is already checked above with `str_starts_with($name, '---')`. So checking again for `-` at 2nd position is redundant.

Reply:
Ah I missed that.